Cargo.toml

```diff
@@ -104,9 +104,9 @@ reqwest = { version = "0.12.7", default-features = false }
 ring = "0.17.13"
 roaring = "0.10.6"
 rustls = { version = "0.23.12", default-features = false }
+rust_decimal = { version = "1.37.1", features = ["macros"] }
```

Comment:
Since you removed the usage of the rust decimal macro is this change still needed?

Reply:
Ah, no it isn't. Updating.

Reply:
Updated